R/serializer.R

```diff
@@ -306,6 +306,22 @@ serializer_parquet <- function(type = "application/vnd.apache.parquet") {
   )
 }
 
+#' @describeIn serializers excel serializer. See also: [writexl::write_xlsx()]
+#' @export
+serializer_excel <- function(..., type = "application/vnd.openxmlformats-officedocument.spreadsheetml.sheet") {
+  if (!requireNamespace("writexl", quietly = TRUE)) {
+    stop("The writexl package is not available but is required in order to use the writexl serializer",
+         call. = FALSE)
+  }
+
+  serializer_write_file(
```

Comment:
Notes for me... `writexl::write_xlsx()` can not leverage the changes in #972 as write_xl is expecting a file string and does not return a raw vector here: https://github.com/ropensci/writexl/blob/7c06c338cdddc0cc1c078afe3173a71a6efe33ff/src/write_xlsx.c#L209

Reply:
bummer